server/models/MigrationLog.ts

```diff
@@ -11,6 +11,7 @@ export enum MigrationLogType {
   BAN_ACCOUNTS = 'BAN_ACCOUNTS',
   MOVE_ORDERS = 'MOVE_ORDERS',
   MOVE_EXPENSES = 'MOVE_EXPENSES',
+  CHECK_SCRIPT = 'CHECK_SCRIPT',
```

Comment:
Maybe not a fan of `CHECK_SCRIPT`.

`CHECK_MODEL`, `MODEL_CHECK`, `MODEL_FIX`, `CHECK_MODEL_FIX`, `MODEL_CHECK_FIX` not sure.

Reply:
What about `DATABASE_CHECK` or `DATABASE_FIX`?

Reply:
Why not `DATABASE_FIX`, _fix_ is better than _check_.

I prefer `MODEL_FIX` because the checks are in the "checks/model" folder and not "checks/database".